### ISSUE TITLE
Crash when save

### ISSUE THREAD

**Issue by andybonnetto** [amathislab/dlc2action_annotation#4]
In niels_bufixes:
GUI crashes when save annotations (CTRL + S or File>Save):
```
   line 37, in form_annotations_csv
      cat_times = times[animal_idx][cat_idx].tolist()
  AttributeError: 'list' object has no attribute 'tolist'
```

**Comment by n-poulsen** [comment]
Thanks for spotting this! Was always numpy arrays for me. I'll add a fix to the pull request.

**Comment by n-poulsen** [comment]
This has been patched in commit [63df320f](https://github.com/amathislab/dlc2action_annotation/pull/3/commits/63df320fb4a9ed110a3064fa0aca39ae2b14a6cc). 

Could you try it and let me know if it fixes your issue?

**Comment by andybonnetto** [comment]
Yes it works fine !
